src/components/application_manager/src/resumption/resume_ctrl_impl.cc

```diff
@@ -545,8 +545,12 @@ void ResumeCtrlImpl::AddCommands(ApplicationSharedPtr application,
         saved_app[strings::application_commands];
     for (size_t i = 0; i < app_commands.length(); ++i) {
       const smart_objects::SmartObject& command = app_commands[i];
+      const uint32_t cmd_id = command[strings::cmd_id].asUInt();
+      const bool is_resumption = true;
 
-      application->AddCommand(command[strings::cmd_id].asUInt(), command);
+      application->AddCommand(cmd_id, command);
+      application->help_prompt_manager().OnVrCommandAdded(
+          cmd_id, command, is_resumption);
```

Comment:
Initial `SetGlobalProperties` requests should be sent after these commands are finished resuming.

Reply:
@jacobkeeler when all commands are finish resuming, then final `SetGlobalProperties` with all these resumed commands will be send. 
We can't see any reason to make a spam of `SetGlobalProperties` requests after resuming every separate command. For example if 30 commands will be resumed, then SDL will send 30 `TTS.SetGlobalProperties` + 30 `UI.SetGlobalProperties` + one last described above. From this point of view it would be much better to send just last one `SetGlobalProperties` with all resumed commands. Isn't it?

Reply:
@AKalinich-Luxoft This was what I was suggesting (sending one SetGlobalProperties after resumption is completed), but I didn't see any logic which would trigger this, which is why I included this comment. I was not suggesting sending this message after each individual command is resumed, sorry if that was unclear.

Reply:
@jacobkeeler `SetGlobalProperties` will be called right after commands resumption. Application commands are resuming in [AddCommands()](https://github.com/smartdevicelink/sdl_core/blob/6ca86cb8ad83b9b75148e067829b9972ea742bb5/src/components/application_manager/src/resumption/resume_ctrl_impl.cc#L476) and `SetGlobalProperties` will be send in [SetGlobalProperties()](https://github.com/smartdevicelink/sdl_core/blob/6ca86cb8ad83b9b75148e067829b9972ea742bb5/src/components/application_manager/src/resumption/resume_ctrl_impl.cc#L478) right after that so it will be triggered anyway.

Reply:
@jacobkeeler if there are no commands for resumption then SDL just will send `SetGlobalProperties` with other resumed parameters only